docs/manage-users/user-credentials.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Manage User Credentials
+sidebar_label: User Credentials
+slug: /user-credentials
+---
+
+You can manage all the Credentials you own under the `Credentials` page of your profile. This article, we'll walk you through how to manage and share the Credentials you own across projects.
+
+### All your Credentials in one place
+
+The `Credentials` page of your `User Settings` allows you to add, view, edit or delete the Credentials you own. It is the central place to manage your Credentials across all projects you collaborate on. 
+
+![User Credential](/img/lightning_user_profile_credentials.png) 
+
+![User Credentials List](/img/lightning_edit_user_credential.png)
+
+For guidance on how to set up a new Credential, head over to our [Manage Credentials](http://localhost:3000/documentation/next/manage-projects/manage-credentials) page.
```

Comment:
@ritazagoni this linking is not right - and shouldn't include the full URL. The link should use the path in directory starting from `/documentation/next/...`. So I'm pretty sure this will work: `/documentation/next/manage-projects/manage-credentials`. Implement it and then test it out or lmk if you want me to test